### PR TITLE
Removes non-determinism from InterpreterUtilTest

### PR DESCRIPTION
**Pull request by lukasz-golebiewski**
## Overview
See description

### JIRA ticket:
Ad-hoc

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: casper/src/test/scala/coop/rchain/casper/util/rholang/InterpreterUtilTest.scala [hunk 1]
```diff
@@ -58,24 +58,25 @@ class InterpreterUtilTest
     )
 
   "computeBlockCheckpoint" should "compute the final post-state of a chain properly" in effectTest {
+    val time = System.currentTimeMillis()
```
Comment (KentShikama) [inline casper/src/test/scala/coop/rchain/casper/util/rholang/InterpreterUtilTest.scala]: Why not use LogicalTime?

**Review by KentShikama — APPROVED** [review]

File: casper/src/test/scala/coop/rchain/casper/util/rholang/InterpreterUtilTest.scala (hunk 1 above)
Comment (lukasz-golebiewski) [reply casper/src/test/scala/coop/rchain/casper/util/rholang/InterpreterUtilTest.scala]: It's in the `comm` layer

Comment (lukasz-golebiewski) [reply casper/src/test/scala/coop/rchain/casper/util/rholang/InterpreterUtilTest.scala]: I'm going to replace `currentTimeMillis` with `0`

**Review by EncodePanda — APPROVED** [review]
https://media.giphy.com/media/LSFXlAmuWhf6KN49FG/giphy.gif

**Pull request merged**